product_pricelist_margins/views/product_pricelist_item_views.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="product_pricelist_item_tree_view_inherit" model="ir.ui.view">
+        <field name="model">product.pricelist.item</field>
+        <field name="inherit_id" ref="product.product_pricelist_item_tree_view" />
+        <field name="arch" type="xml">
+            <field name="price" position="after">
+                <field name="cost" />
+                <field name="margin" />
+                <field name="margin_percent" />
```

Comment:
```suggestion
                <field name="margin" optional="show"/>
                <field name="margin_percent" optional="show"/>
```

Reply:
allow users to hide percent and / or net value if they don't need the information.